teckers/Podfile.lock

```diff
@@ -1,79 +1,96 @@
 PODS:
-  - Alamofire (4.8.2)
```

Comment:
Make sure this is the latest podfile version in order to merge this PR

Reply:
ok, I will check it